src/ToWords.ts

```diff
@@ -79,9 +93,19 @@ export class ToWords {
     return this.locale;
   }
 
-  public convert(number: number, options: ConverterOptions = {}): string {
+  /**
+   *
+   * @param number - user input number to be converted
+   * @param options - configuration set by user
```

Comment:
number - The number to be converted into the words.
options - Converter Options object.